src/picknik_ur_studio_integration_testing/test/test_basic_ur5e_objectives.cpp

```diff
@@ -0,0 +1,232 @@
+  pose_name_parameter.pose_value = grasp_pose;
+
+  do_objective_goal->parameter_overrides.insert(do_objective_goal->parameter_overrides.end(), { pose_name_parameter });
+
+  EXPECT_TRUE(sendDoObjectiveSequenceGoal(std::move(do_objective_goal), 30.0));
+}
+
+TEST_F(ObjectiveFixture, DISABLED_TestInspectSurface)
+{
+  using SetTransform = moveit_studio_agent_msgs::srv::SetTransform;
+
+  ASSERT_TRUE(setupDoObjectiveSequenceClient());
+
+  const auto set_transform_client = node_->create_client<SetTransform>("/set_transform");
+
+  const auto transform = [] {
+    TransformStamped msg;
+    msg.header.frame_id = "world";
+    msg.child_frame_id = "selected_frame";
+    msg.transform.translation = geometry_msgs::build<Vector3>().x(0.48976525).y(-0.1367085).z(0.27501845);
+    msg.transform.rotation = geometry_msgs::build<Quaternion>().x(0.7063694).y(0.44723025).z(0.29349735).w(0.46355873);
+    return msg;
+  }();
+  const auto set_transform_req =
+      std::make_shared<SetTransform::Request>(moveit_studio_agent_msgs::build<SetTransform::Request>()
+                                                  .action(SetTransform::Request::SET_OR_UPDATE)
+                                                  .tform(transform));
+
+  auto set_transform_future = set_transform_client->async_send_request(set_transform_req);
+  ASSERT_EQ(set_transform_future.wait_for(std::chrono::seconds(10)), std::future_status::ready);
+  auto set_transform_result = set_transform_future.get();
+  ASSERT_TRUE(set_transform_result->success);
+
+  auto do_objective_goal = std::make_unique<DoObjectiveSequence::Goal>();
+  do_objective_goal->objective_name = "Move to Target";
+
+  EXPECT_TRUE(sendDoObjectiveSequenceGoal(std::move(do_objective_goal), 30.0));
+}
+
+TEST_F(ObjectiveFixture, TestInterpolateToJointState)
+{
+  ASSERT_TRUE(setupDoObjectiveSequenceClient());
+
+  auto do_objective_goal = std::make_unique<DoObjectiveSequence::Goal>();
+  do_objective_goal->objective_name = "Interpolate to Joint State";
+
+  // Update the joint_state parameter used when planning the objective
+  BehaviorParameter pose_name_parameter;
+  pose_name_parameter.behavior_namespaces = { "interpolate_to_joint_state" };
+  pose_name_parameter.description.name = "target_joint_state";
+  pose_name_parameter.description.type = BehaviorParameterDescription::TYPE_JOINT_STATE;
+  pose_name_parameter.joint_state_value.name = { "shoulder_pan_joint", "shoulder_lift_joint", "elbow_joint",
+                                                 "wrist_1_joint",      "wrist_2_joint",       "wrist_3_joint" };
+  pose_name_parameter.joint_state_value.position = { 0, -0.78, -0.51, -1.57, 1.65, 0 };
+
+  do_objective_goal->parameter_overrides = { pose_name_parameter };
+
+  EXPECT_TRUE(sendDoObjectiveSequenceGoal(std::move(do_objective_goal), 30.0));
+}
+
+// TODO: Enable once admittance control works with Ignition Gazebo, or we can override behaviors for sim configs.
```

Comment:
Since we're not using Gazebo, and it's not called Ignition anymore, maybe reword to e.g.

```suggestion
// TODO: Enable once admittance control works in the current simulation environment, or we can override behaviors for sim configs.
```

... although this should work now because overriding Objectives for sim configs should work.